doc-content/enterprise-only/admin-and-config/kie-server-smart-router-enable-tls-support-proc.adoc

```diff
@@ -14,15 +14,38 @@ xref:clustering-smart-router-install-proc_clustering-runtime-standalone[].
 endif::[]
 
 .Procedure
-* To start Smart Router with TLS support and HTTPS enabled, use the TLS keystore properties, for example:
+Use one of the following methods to start Smart Router:
```

Comment:
[style] This was brought up during our latest workshop. When handling nonsequential actions, what is the best phrasing? Example I've seen are:

"To do this, perform one of the following tasks:" (The one I thought we use)
"To do this, complete one of the following:" (misses the following subject)
"Use one of the following methods to do this:"

Example from *Handling nonsequential actions* - IBMSG p.88

----
Click *Configure* to do any of the following tasks:
* Choice 1
* Choice 2
----

I thought we use "To do this, perform one of the following tasks:"